src/PostRector/Rector/UnusedImportRemovingPostRector.php

```diff
@@ -36,10 +34,6 @@ public function enterNode(Node $node): ?Node
             return null;
         }
 
-        if (! SimpleParameterProvider::provideBoolParameter(Option::REMOVE_UNUSED_IMPORTS)) {
```

Comment:
This check is not needed, if this option is not set, this post rector will no be run